### PR TITLE
Converted PropertiesTest and TraitsTest to Kotlin

### PR DESCRIPTION
**Pull request by Jessica-Kuo**

### CODE REVIEW

**Comment by prayansh** [comment]
Since we r migrating tests, one good thing to add here would be screenshots showing that coverage hasnt decreased. Maybe like a before and after state.

**Comment by Jessica-Kuo** [comment]
Right, that would be a quick way to show the tests produce the same results. I'll get screenshots of the test results from running the tests in Java and then running them with these files.

**Comment by migs647** [comment]
Also, lets create a new Pull Request so we don't keep adding changes building the changes up that have already been merged.

File: build.gradle [hunk 1]
```diff
@@ -1,5 +1,6 @@
 buildscript {
-  repositories {
+    ext.kotlin_version = '1.3.72'
```
Comment (prayansh) [inline build.gradle:2]: can we move this here? https://github.com/segmentio/analytics-android/blob/master/build.gradle#L47

would make sense to have all the configs together

**Pull request closed without merging**